docker-compose.yml

```diff
@@ -35,10 +35,9 @@ services:
     ports:
```

Comment:
When entering the docker-compose up command, this microservice does not start, because the required Dockerfile is not specified.